bitnami/external-dns/templates/dep-ds.yaml

```diff
@@ -12,7 +12,8 @@ kind: Deployment
 metadata:
   name: {{ template "external-dns.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
-  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- $podLabels := merge .Values.podLabels .Values.commonLabels }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" $podLabels "context" $ ) | nindent 4 }}
```

Comment:
Please note that these aren't pod labels but sts/deployment labels. Therefore, we shouldn't apply `.Values.podLabels` here

Reply:
Ah you are completely right
i was fooled by the helmfile diff which told me the labels were removed. but they were just removed from the deployment